DownloadURL/explainer.md

```diff
@@ -0,0 +1,203 @@
+# Explainer: Drag Multiple Virtual Files Out of Browser
+
+Author: [Joone Hur](https://github.com/joone)
+
+# Participate
+
+- [Issue tracker](https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/DownloadURL)
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Table of Contents
+
+- [Introduction](#introduction)
+- [Goals](#goals)
+- [Non-goals](#non-goals)
+- [Problem: The Single-File Limitation](#problem-the-single-file-limitation)
+  - [Example of Current Behavior](#example-of-current-behavior)
+  - [Developer Workarounds and Limitations](#developer-workarounds-and-limitations)
+    - [Using an array](#using-an-array)
+    - [Using newline-delimiters](#using-newline-delimiters)
+  - [UX perspective](#ux-perspective)
+- [Proposed Solution: Extending the `DownloadURL` Data Format](#proposed-solution-extending-the-downloadurl-data-format)
+  - [New `DownloadURL` Data Format](#new-downloadurl-data-format)
+  - [Why JSON?](#why-json)
+- [Backward Compatibility](#backward-compatibility)
+- [Future Considerations](#future-considerations)
+- [Security & UX Considerations](#security--ux-considerations)
+  - [Drag Bomb](#drag-bomb)
+  - [Limit Download Requests](#limit-download-requests)
+  - [Single-click button to delete all downloaded files](#single-click-button-to-delete-all-downloaded-files)
+- [Acknowledgements](#acknowledgements)
+- [Referenecs](#referenecs)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Introduction
+
+Chromium has supported a non-standard drag type (`DownloadURL`) that allows Windows users to drag a virtual file out of the browser. However, the `DownloadURL` drag type is limited to supporting only a single file per drag-and-drop operation. To support more robust workflows, we propose extending the DownloadURL drag type to allow multiple files to be dragged simultaneously.
+
+# Goals
+
+Enable users to drag multiple files from Chromium to a desktop folder on Windows using an extended `DownloadURL` drag type.
+
+# Non-goals
+This proposal does not aim to:
+* Enable drag-and-drop of files into other browser contexts, whether between the same browser or different browsers.
+* Standardize this behavior across browsers via W3C. The proposal is specific to Chromium and Windows environments.
```

Comment:
Is there history you can share on why this hasn't been standardized already?

Reply:
I’ve included this history in the updated explainer.